content/cosign-2-0-released.md

```diff
@@ -0,0 +1,43 @@
+    * You must also include `--insecure-ignore-tlog=true` when verifying an artifact that was not uploaded to Rekor.
+    * Examples of when you may want to skip uploading to the transparency log are if you have a private Sigstore deployment that does not use transparency or a private artifact.
+    * We strongly encourage all other use-cases to upload artifact signatures to Rekor. Transparency is a critical component of supply chain security, to allow artifact maintainers and consumers to monitor a public log for their artifacts and signing identities.
+* Verification now requires identity flags, `--certificate-identity` and `--certificate-oidc-issuer`. Like verifying a signature with a public key, it's critical to specify who you trust to generate a signature for identity-based signing. See https://github.com/sigstore/cosign/issues/2056 for more discussion on this change.
+* `--certificate-email` has been removed. Use `--certificate-identity`, which supports not only email verification but also any identity specified in a certificate, including SPIFFE, GitHub Actions, or service account identities. 
+* Cosign no longer supports providing a certificate that does not conform to the Fulcio certificate profile, which includes setting the `SubjectAlternativeName` and OIDC Issuer OID. To verify with a non-conformant certificate, extract the public key from the certificate and verify with `cosign verify --key <key.pem>`.  We are actively working on more support for custom certificates for those who want to bring their existing PKI.
+* Signing OCI images by tag prints a warning and is strongly discouraged, e.g. `cosign sign container.registry.io/foo:tag`. This is considered insecure since tags are mutable. If you want to specify a particular image, you are recommended to do so by digest.
+* SCT verification, a proof of inclusion in a certificate transparency log, is now on by default for verifying Fulcio certificates. For private deployments without certificate transparency, use `--insecure-ignore-sct=true` to skip this check.
+* DSSE support in `verify-blob` has been removed. You can now verify attestations using `verify-blob-attestation`.
+* Environment variable `SIGSTORE_TRUST_REKOR_API_PUBLIC_KEY` has been removed. For private deployments, if you would like to set the Rekor public key to verify transparency log entries, use either a [TUF setup](https://blog.sigstore.dev/sigstore-bring-your-own-stuf-with-tuf-40febfd2badd) or set `SIGSTORE_REKOR_PUBLIC_KEY` with the PEM of the custom Rekor public key..
+* `verify-blob` no longer searches for a certificate. You must provide one with either `--certificate` or `--bundle`.
+* `cosign attest --type {custom|vuln}` (and `cosign verify-attestation`) will now use the RFC 3986 compliant URIs, adding `https://`, so that these predicate types are compliant with the in-toto specification.
+
```

Comment:
This is probably worth pointing out as well? Or maybe I missed it?
https://github.com/sigstore/cosign/pull/2718

Reply:
That's the last bullet point here, about being compliant with RFC 3986

Reply:
I don't think so, that PR changes the two (spdx/cyclonedx) attestations so they are no longer wrapped. It matters if you are trying to take the attestation and do something with it?
https://github.com/sigstore/cosign/pull/2718/files#diff-9e762d0df1502ee31a1de1a3bc67be6c2707ec3e6117be3147907a60ef5fb854L263

Reply:
Ah yep, let me update that!

Reply:
Updated!